goclean.sh

```diff
@@ -5,14 +5,19 @@
 # 3. golint        (https://github.com/golang/lint)
 # 4. go vet        (http://golang.org/cmd/vet)
 # 5. test coverage (http://blog.golang.org/cover)
+#
+# gometaling (github.com/alecthomas/gometalinter) is used to run each each
+# static checker.
 
 set -e
 
 # Automatic checks
 test -z "$(gofmt -l -w .     | tee /dev/stderr)"
-test -z "$(goimports -l -w . | tee /dev/stderr)"
-test -z "$(golint .          | tee /dev/stderr)"
-go vet ./...
+test -z "$(gometalinter --disable-all \
+--enable=golint \
+--enable=vet \
+--enable=goimports \
+--deadline=20s ./... | grep -v 'ALL_CAPS\|OP_' 2>&1 | tee /dev/stderr)"
```

Comment:
The grep isn't needed here.  That us a workaround for `txscript` which doesn't apply in `btcrpcclient`.